### PR TITLE
chore: fix eslint ignore patterns for vite

### PR DESCRIPTION
**Pull request by lifeiscontent**
Closes: #9068

- [ ] Docs
- [ ] Tests

Testing Strategy:

### CODE REVIEW

**Comment by lifeiscontent** [comment]
@brophdawg11 here's the fixes we talked about earlier today 😄

File: templates/vite-cloudflare/.eslintrc.cjs [hunk 1]
```diff
@@ -22,6 +22,7 @@ module.exports = {
 
   // Base config
   extends: ["eslint:recommended"],
+  ignorePatterns: ["!**/.server", "!**/.client"],
```
Comment (brophdawg11) [inline templates/vite-cloudflare/.eslintrc.cjs:25]: ignorePatterns: ["!**/.server", "!**/.client"],
```suggestion
  ignorePatterns: ["!**/.server", "!**/.client"],

  // Base config
  extends: ["eslint:recommended"],
```

File: templates/vite-express/.eslintrc.cjs [hunk 2]
```diff
@@ -22,6 +22,7 @@ module.exports = {
 
   // Base config
   extends: ["eslint:recommended"],
+  ignorePatterns: ["!**/.server", "!**/.client"],
```
Comment (brophdawg11) [inline templates/vite-express/.eslintrc.cjs:25]:
```suggestion
  ignorePatterns: ["!**/.server", "!**/.client"],

  // Base config
  extends: ["eslint:recommended"],
```

File: templates/vite/.eslintrc.cjs [hunk 3]
```diff
@@ -22,6 +22,7 @@ module.exports = {
 
   // Base config
   extends: ["eslint:recommended"],
+  ignorePatterns: ["!**/.server", "!**/.client"],
```
Comment (brophdawg11) [inline templates/vite/.eslintrc.cjs:25]:
```suggestion
  ignorePatterns: ["!**/.server", "!**/.client"],

  // Base config
  extends: ["eslint:recommended"],
```

**Pull request merged**